### PR TITLE
Tyler Ren's card

### PR DESCRIPTION
**Pull request by rentianle2023**
Added my card inforamtion to index.html

### CODE REVIEW

**Review by CandidDeer — CHANGES_REQUESTED** [review]
Please fix the following issues: target should remain "_blank", whereas the links need to be stated in href="". thx.

File: index.html [hunk 1]
```diff
@@ -727,6 +727,28 @@ <h2>A project for first-time contributions</h2>
...
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/rentianle2023" target="_blank">TylerRen</a>
+          </p>
+          <p class="about">I'm a self-taught backend software engineer current working at Citibank Shanghai</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="https://stackoverflow.com/" title="Stack Overflow">Stack Overflow</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:741]:
```suggestion
                <a href="https://stackoverflow.com/" target="_blank" title="Stack Overflow">Stack Overflow</a>
```

File: index.html [hunk 2]
```diff
@@ -727,6 +727,28 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="https://leetcode.com/" title="Leetcode">Leetcode</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:744]:
```suggestion
                <a href="https://leetcode.com/" target="_blank" title="Leetcode">Leetcode</a>
```

File: index.html [hunk 3]
```diff
@@ -727,6 +727,28 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="https://www.freecodecamp.org/" title="freeCodeCamp">freeCodeCamp</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:747]:
```suggestion
                <a href="https://www.freecodecamp.org/" target="_blank" title="freeCodeCamp">freeCodeCamp</a>
```

**Review by StackOverflowIsBetterThanAnyAI — COMMENTED** [review]
I highlighted what has been said in the previous comment.

**Pull request closed without merging**